### PR TITLE
Adjust the spacing of the new Tudor crown

### PR DESCRIPTION
**Pull request by MartinJJones**
Adjust the spacing of the new Tudor crown to closely match the new SVG Tudor crown to be released in 5.1.

### 4.8 - Before

Screenshot
localhost_3000_components_header_with-Tudor-crown_preview-before

Video comparison, switching from 4.8 to 5.1

https://github.com/alphagov/govuk-frontend/assets/28779939/086385b6-9984-4bfa-a834-0dbe493e8ead

### 4.8 - After

Screenshot
localhost_3000_components_header_with-Tudor-crown_preview-after

Video comparison, switching from 4.8 to 5.1

https://github.com/alphagov/govuk-frontend/assets/28779939/a73eaf53-49e6-47b2-9b6d-8581e625443d

### CODE REVIEW

**Comment by querkmachine** [comment]
For completion's sake, this PR's header with a product name.

Compared with the 5.1 lockup with product name. Slight difference in logotype size but this is to be expected, as the text is slightly different in the new lockup (or subpixel rounding makes it that way, anyway).

Animation comparing this PR with the 5.1 logo with product name

In Chromium:
Logo with product name in Chromium

In Safari:
Logo with product name in Safari

In Firefox:
Logo with product name in Firefox

Overall this seems like a sensible change to me.

**Comment by querkmachine** [comment]
We probably want this tagging onto the end of the existing changelog entry before I put approval on it. You can do this if you want, or someone on the team can handle it otherwise. 

Then it'll need a second approval before merging.

**Comment by MartinJJones** [comment]
> We probably want this tagging onto the end of the existing changelog entry before I put approval on it. You can do this if you want, or someone on the team can handle it otherwise.

Thanks, I've just added a changelog entry to reference this PR, hope this is OK

File: CHANGELOG.md
Comment (querkmachine) [inline CHANGELOG.md:17]:
```suggestion
- [#4677: Adjust the spacing of the new Tudor crown (v4.x)](https://github.com/alphagov/govuk-frontend/pull/4677) - thanks to [Martin Jones](https://github.com/MartinJJones) for contributing this change
```

Credit where it's due 😉

**Review by querkmachine — APPROVED** [review]
Thanks for noticing and handling this!

File: CHANGELOG.md
Comment (MartinJJones) [reply CHANGELOG.md]: Awesome, thanks very much! 😃 

Hope it's OK, I also added a mention for Mónica Crusellas, the interaction Designer on our team who helped me with this change - https://github.com/alphagov/govuk-frontend/pull/4677/commits/f607a831661319a24d9de85700ce29851e695230

**Comment by MartinJJones** [comment]
> Thanks for noticing and handling this!

No problem, thanks very much for the reviewing so quickly.

Do I also need to request a 2nd review before merging? thanks

File: CHANGELOG.md
Comment (querkmachine) [reply CHANGELOG.md]: Good with me!

**Comment by querkmachine** [comment]
@MartinJJones Yep, just need someone else on the team to review it and we should be good. 

The same tweaks are probably worth doing on the `support/3.x` branch too, but we can probably handle that now we're confident of the changes here.

**Review by christopherthomasdesign — APPROVED** [review]
Looks good to me, thanks so much for spotting and sorting this @MartinJJones !

**Comment by MartinJJones** [comment]
Thanks again for reviewing the changes, much appreciated 😃

**Pull request merged**
